README.md

```diff
@@ -0,0 +1,147 @@
+
+
+# VyOS Collection
+[![CI](https://zuul-ci.org/gated.svg)](https://dashboard.zuul.ansible.com/t/ansible/project/github.com/ansible-collections/vyos) <!--[![Codecov](https://img.shields.io/codecov/c/github/ansible-collections/vyos)](https://codecov.io/gh/ansible-collections/vyos)-->
+
+The Ansible VyOS collection includes a variety of Ansible content to help automate the management of VyOS network appliances.
+
+This collection has been tested against VyOS 1.1.8 (helium).
+
+
+### Supported connections
+The VyOS collection supports ``network_cli`` connections.
+
+## Included content
+
+Click the ``Content`` button to see the list of content included in this collection.
+
+## Installing this collection
+
+You can install the VyOS collection with the Ansible Galaxy CLI:
+
+    ansible-galaxy collection install vyos.vyos
+
+You can also include it in a `requirements.yml` file and install it with `ansible-galaxy collection install -r requirements.yml`, using the format:
+
+```yaml
+---
+collections:
+  - name: vyos.vyos
+    version: 0.0.2
+```
+## Using this collection
+
+
+This collection includes [network resource modules](https://docs.ansible.com/ansible/latest/network/user_guide/network_resource_modules.html).
+
+### Using modules from the VyOS collection in your playbooks
+
+You can either call modules by their Fully Qualified Collection Namespace (FQCN), like `vyos.vyos.vyos_l3_interfaces`, or you can call modules by their short name if you list the `vyos.vyos` collection in the playbook's `collections`, as follows:
+
+```yaml
+---
+- hosts: vyos01
+  gather_facts: false
+  connection: network_cli
+
+  collections:
```

Comment:
collections keyword is not required if FQCN is used in task name. Probably can be split using collections keyword and only task name and other without collection name and FQCN task name.